3.Tests/UI/StepDefinitions/Settings/Profile/ViewProfileStepDefinition.cs

```diff
@@ -0,0 +1,61 @@
+﻿using TechTalk.SpecFlow;
+using Todoly.Core.Helpers;
+using Todoly.Tests.UI.Steps.Commons;
+using Todoly.Views.WebAppPages;
+
+namespace SeleniumTest.Tests;
+
+[Binding]
+[Scope(Feature = "Profile settings view")]
+public class ViewProfileStepDefinition : CommonSteps
+{
+    private readonly HomePage _homePage;
+    private ProfilePage? _profilePage;
+    private readonly ScenarioContext _scenarioContext;
+    public ViewProfileStepDefinition(ScenarioContext scenarioContext) : base(scenarioContext)
+    {
+        _homePage = new HomePage();
+        _scenarioContext = scenarioContext;
+    }
+
+    [When(@"the user clicks on the Settings option on the Nav Bar")]
+    public void WhentheuserclicksontheSettingsoptionontheNavBar()
+    {
+        _homePage.SettingsButton.Click();
+        _profilePage = new ProfilePage();
+    }
+
+    [Then(@"the Profile Settings view should be opened")]
+    public void ThentheProfileSettingsviewshouldbeopened()
+    {
+        Assert.True(_profilePage!.CloseButton.WebElement.Displayed);
+    }
+
+    [Then(@"the Full Name label and input should be displayed")]
+    public void ThentheFullNamelabelandinputshouldbedisplayed()
+    {
+        Assert.True(_profilePage!.FullNameTextField.WebElement.Displayed);
+        string fullname = ConfigBuilder.Instance.GetString("api", "UserFullName");
+        Assert.That(_profilePage!.FullNameTextField.WebElement.GetAttribute("value"), Is.EqualTo(fullname));
+    }
+
+    [Then(@"the Email label and input should be displayed")]
+    public void ThentheEmaillabelandinputshouldbedisplayed()
+    {
+        Assert.True(_profilePage!.EmailTextField.WebElement.Displayed);
+        string email = ConfigBuilder.Instance.GetString("TODO-LY-EMAIL");
+        Assert.That(_profilePage!.EmailTextField.WebElement.GetAttribute("value"), Is.EqualTo(email));
+    }
+
+    [Then(@"the Old Password label and input should be displayed")]
+    public void ThentheOldPasswordlabelandinputshouldbedisplayed()
+    {
+        Assert.True(_profilePage!.OldPassTextField.WebElement.Displayed);
+    }
+
+    [Then(@"the New Password label and input should be displayed")]
+    public void ThentheNewPasswordlabelandinputshouldbedisplayed()
+    {
+        Assert.True(_profilePage!.NewPassTextField.WebElement.Displayed);
```

Comment:
You are starting to have duplicated step implementations, lines 31, 53 and 59 for example

Reply:
You're right. Could it be a possible solution to have a method inside the ProfilePage view that asserts that? Receiving the element as a parameter for example

Reply:
Or actually I could create a common step like I did with the credentials, right?

Reply:
Like this:
[Then(@"the (New Password|Old Password|Email) label and input should be displayed")]
public void ValidateDisplay(WebElement element)
{
        Assert.True(element.Displayed);
}